usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -18,6 +18,12 @@ def add_arguments(self, parser):
         parser.add_argument('file', nargs=1, help='the file to load')
 
     def handle(self, *args, **options):
+        # Create a new submission attributes object for this timestamp
+        subattr = SubmissionAttributes()
```

Comment:
We just need to review the ETL process to make sure updating records is smooth. It _should_ do it right now but I'm not too confident about it.